obspy/io/nordic/core.py

```diff
@@ -466,6 +474,50 @@ def _read_uncertainty(tagged_lines, event):
     return event
 
 
+def _read_highaccuracy(tagged_lines, event):
+    """
+    Read high accuracy origin line.
+    """
+    if 'H' not in tagged_lines.keys():
+        return event
+    # In principle there shouldn't be more than one high precision line
+    line = tagged_lines['H'][0][0]
+    try:
+        dt = {'Y':  int(line[1:5]),  'MO': int(line[6:8]),
+              'D':  int(line[8:10]),  'H': int(line[11:13]),
+              'MI': int(line[13:15]), 'S': float(line[16:23])}
+    except ValueError:
+        pass
```

Comment:
It doesn't look like this should be a plain pass - maybe a `dt = dict()`, then the following `try:.. expect:` would be an `if` - there is potential for the next `try:.. expect` to catch other, unexpected `ValueError`s.

Reply:
I now use your _int_conv routines